sporkfish/engine.py

```diff
@@ -39,8 +46,32 @@ def best_move(self, board: Board, timeout: Optional[float] = None) -> chess.Move
         :return: The selected chess move.
         :rtype: chess.Move
         """
-        opening_move = self._opening_book.query(board)
-        return opening_move or self._searcher.search(board, timeout)[1]
+        if opening_move := self._opening_book.query(board):
+            return opening_move
+
+        end_move = self._use_endgame_tablebase(board)
```

Comment:
One thing I remembered, think this should check if the reader is open - shouldn't do the 64 square checking every time

Reply:
Ahha, so there's gonna be some if-else condition inside this method?

Reply:
Probably easier to move the check into the query function imo. We don't (or shouldn't) have visibility to the _db member of `EndgameTablebase`

Reply:
Done.